adagrams/game.py

```diff
@@ -1,13 +1,92 @@
 from random import randint
 
+LETTER_POOL_DISTRIBUTION = {
+    'A': 9, 'B': 2, 'C': 2, 'D': 4, 'E': 12, 'F': 2, 'G': 3, 'H': 2,
+    'I': 9, 'J': 1, 'K': 1, 'L': 4, 'M': 2, 'N': 6, 'O': 8, 'P': 2,
+    'Q': 1, 'R': 6, 'S': 4, 'T': 6, 'U': 4, 'V': 2, 'W': 2, 'X': 1,
+    'Y': 2, 'Z': 1
+}
+
+SCORES = {
+        "A": 1, "E": 1, "I": 1, "O": 1, "U": 1, "L": 1, "N": 1, "R": 1, "S": 1, "T": 1,
+    "D": 2, "G": 2, "B": 3, "C": 3, "M": 3, "P": 3, "F": 4, "H": 4, "V": 4, "W": 4, "Y": 4,
+    "K": 5, "J": 8, "X": 8, "Q": 10, "Z": 10
+    }
+
+LETTER_POOL_LIST = []
+for letter, quantity in LETTER_POOL_DISTRIBUTION.items():
+    for _ in range (quantity):
+        LETTER_POOL_LIST.append(letter) 
+
+
 def draw_letters():
-    pass
+    copy_list = list(LETTER_POOL_LIST)
```

Comment:
Note that if the `LETTER_POOL_LIST` was created inside of this function then you wouldn't need to create a copy. The list would be freshly made each time the function was called. Another reason to move it inside of the function is that it will be the only function to utilize it. Having it locally scoped to the function will protect it from being mutated by other functions.